shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -495,22 +492,21 @@ - (void)pushRoute:(NSString*)route {
 }
 
 - (void)loadView {
-  self.view = GetViewOrPlaceholder(_flutterView.get());
+  self.view = GetViewOrPlaceholder(self.flutterView);
   self.view.multipleTouchEnabled = YES;
   self.view.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
 
   [self installSplashScreenViewIfNecessary];
-  UIScrollView* scrollView = [[UIScrollView alloc] init];
-  scrollView.autoresizingMask = UIViewAutoresizingFlexibleWidth;
+  self.scrollView = [[UIScrollView alloc] init];
+  self.scrollView.autoresizingMask = UIViewAutoresizingFlexibleWidth;
   // The color shouldn't matter since it is offscreen.
-  scrollView.backgroundColor = UIColor.whiteColor;
-  scrollView.delegate = self;
+  self.scrollView.backgroundColor = UIColor.whiteColor;
+  self.scrollView.delegate = self;
   // This is an arbitrary small size.
-  scrollView.contentSize = CGSizeMake(kScrollViewContentSize, kScrollViewContentSize);
+  self.scrollView.contentSize = CGSizeMake(kScrollViewContentSize, kScrollViewContentSize);
   // This is an arbitrary offset that is not CGPointZero.
-  scrollView.contentOffset = CGPointMake(kScrollViewContentSize, kScrollViewContentSize);
-  [self.view addSubview:scrollView];
-  _scrollView.reset(scrollView);
+  self.scrollView.contentOffset = CGPointMake(kScrollViewContentSize, kScrollViewContentSize);
```

Comment:
Previously we created a local, set all its properties, then assigned to self.scrollView at the bottom. This reads fine and we're still saving the addSubview call for the end.

Reply:
Is there a possibility that it mattered previously the subview was added before the `scrollView` property was set?

Reply:
Probably not, but agree it's worth being paranoid. Reverting that bit of it.